l10n_th_vat_report/static/src/js/l10n_th_vat_report_widgets.js

```diff
@@ -0,0 +1,86 @@
+odoo.define('l10n_th_vat_report.l10n_th_vat_report_widget', function
```

Comment:
Can we remove this file?

Reply:
We can't remove because I use file for actions.